WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/GutenbergLayoutPickerViewController.swift

```diff
@@ -0,0 +1,193 @@
+            subtitleToCategoryBarSpacing.constant +
+            categoryBar.frame.height +
+            maxHeaderBottomSpacing.constant
+    }
+    private var minHeaderHeight: CGFloat {
+        return categoryBar.frame.height + minHeaderBottomSpacing.constant
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        closeButton.setImage(UIImage.gridicon(.crossSmall), for: .normal)
+        styleButtons()
+        layoutHeader()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        (navigationController as? GutenbergLightNavigationController)?.shadowIsHidden = true
+        super.viewWillAppear(animated)
+    }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        (navigationController as? GutenbergLightNavigationController)?.shadowIsHidden = false
+        super.viewDidDisappear(animated)
+    }
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        (navigationController as? GutenbergLightNavigationController)?.shadowIsHidden = false
+        super.prepare(for: segue, sender: sender)
+    }
+
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        if #available(iOS 13.0, *) {
+            if traitCollection.hasDifferentColorAppearance(comparedTo: previousTraitCollection) {
+                styleButtons()
+            }
+        }
+
+        if previousTraitCollection?.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory {
+            styleButtons()
+            layoutHeader()
+        }
+    }
+
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        coordinator.animateAlongsideTransition(in: nil, animation: nil) { (_) in
+            self.scrollViewDidEndDragging(self.tableView, willDecelerate: false)
+        }
+    }
+
+    @IBAction func closeModal(_ sender: Any) {
+        dismiss(animated: true, completion: nil)
+    }
+
+    @IBAction func createBlankPage(_ sender: Any) {
+        createPage(nil)
+    }
+
+    private func createPage(_ template: String?) {
+        guard let completion = completion else {
+            dismiss(animated: true, completion: nil)
+            return
+        }
+
+        dismiss(animated: true) {
+            completion(template)
+        }
+    }
+
+    private func styleButtons() {
+        let seperator: UIColor
+        if #available(iOS 13.0, *) {
+            seperator = .separator
+        } else {
+            seperator = .lightGray
+        }
+
+        [createBlankPageBtn].forEach { (button) in
+            button?.titleLabel?.font = WPStyleGuide.fontForTextStyle(.body, fontWeight: .medium)
+            button?.layer.borderColor = seperator.cgColor
+            button?.layer.borderWidth = 1
+            button?.layer.cornerRadius = 8
+        }
```

Comment:
Asking out of curiosity:
Why did you decide to use `forEach` in this way?

Reply:
The designs have one other button that will need to be added that needs the same styling applied. I built that additional button out at one point so I could test the animations when pushing another ViewController. So I just left this syntax in place for when that button is added back. 

I also considered using a function like `stylePlainFooterButton` instead of doing a `forEach` but didn't feel like a function added any more clarity in this case.

Reply:
Thanks for the explanation, sounds good! :) 

Another alternative I like to use is creating a `styledButtons` OutletCollection (or any other name).
The styling code would be almost the same. It just avoids the array literal in code, and avoids the need to add new elements in code.
Then you just add the buttons you want to style to that collection via ctrl-drag and it gets styled 👍 

But up to you, this is perfectly fine!

Reply:
>Another alternative I like to use is creating a `styledButtons` OutletCollection ...

That is a cool idea! I re-style the buttons for trait changes (dark mode) and content size changes (text) so I think I'll leave it as is, otherwise, I probably would switch to that because it sounds like a neat style.